kclvm/tools/src/LSP/src/util.rs

```diff
@@ -894,6 +895,11 @@ fn line_to_words(text: String) -> HashMap<String, Vec<Word>> {
     result
 }
 
+pub fn is_valid_kcl_name(name: String) -> bool {
```

Comment:
U can add this function into sema/info module.

Reply:
thanks, done